cloudinit/cmd/main.py

```diff
@@ -764,7 +763,8 @@ def status_wrapper(name, args):
 
     v1 = status["v1"]
     v1["stage"] = mode
-    v1[mode]["start"] = time.time()
+    uptime = util.uptime()
+    v1[mode]["start"] = float(uptime)
```

Comment:
I had assumed that this field was used to compare against actual timestamps to know when things happened rather than just tracking elapsed time. Maybe that's a bad assumption, though.

Thoughts?

Reply:
> I had assumed that this field (along with finished) was used to compare against actual timestamps to know when things happened rather than just tracking elapsed time. Maybe that's a bad assumption, though.

The spot that I see it used is in `status.py` which uses the relative time between the stages, which is why `time.monotonic()` won't work.

I didn't find timestamps compared anywhere, and if the code was doing that with `time.time()`, such an implementation would also be susceptible to time change. With uptime in this field it can actually be used to manually correlate with events reported in dmesg / systemd logs / etc, so I think the new code is more correct than it was before